src/arch/x86/mod.rs

```diff
@@ -10,15 +10,12 @@ pub mod reg;
 /// Check out the [module level docs](../index.html#whats-with-regidimpl) for
 /// more info about the `RegIdImpl` type parameter.
 #[allow(non_camel_case_types)]
-pub enum X86_64_SSE<RegIdImpl: RegId> {
-    #[doc(hidden)]
-    _Marker(core::marker::PhantomData<RegIdImpl>),
-}
+pub enum X86_64_SSE {}
 
-impl<RegIdImpl: RegId> Arch for X86_64_SSE<RegIdImpl> {
+impl Arch for X86_64_SSE {
```

Comment:
Please leave the generic parameter in-place. Instead, set `RegIdImpl = reg::id::X86_64CoreRegId` by default.

As per the last paragraph in https://docs.rs/gdbstub/0.4.0/gdbstub/arch/index.html#whats-with-regidimpl, this will allow me to make a single breaking change after all missing `RegId` implementations have been resolved.

Reply:
Sorry, I missed that part in the document. Fixed.